src/activities/hangman/hangman.js

```diff
@@ -136,6 +137,9 @@ function processKeyPress(text) {
     // Add the character to the already typed characters
     alreadyTypedLetters.push(text);
 
+    // add
+    items.guessedText.text=items.guessedText.text.concat(text, ", ");
```

Comment:
it's quite tricky but I would keep all the letters and rebuild the typed letters string using:
items.guessedText.text = qsTr("Attempted: %1").arg(usedLetters)
usedLetters can probably be built using alreadyTypedLetters and join (something like qsTr("Attempted: %1").arg(alreadyTypedLetters.join(", ")).
For translation purpose, it's better to have the whole context (we could have a language that would need to display it on the other direction: "used letters: Attempted" and if we do like now, we won't be able to do it as the code supposes that the letters should be displayed on right of "Attempted: ")